src/olympia/editors/tests/test_views.py

```diff
@@ -2827,6 +2955,44 @@ def test_confirm_auto_approval_no_permission(self):
         assert ActivityLog.objects.filter(
             action=amo.LOG.CONFIRM_AUTO_APPROVED.id).count() == 0
 
+    def test_attempt_to_use_content_review_permission_for_post_review_actions(
+            self):
+        # Try to use confirm_auto_approved outside of content review, while
+        # only having Addons:ContentReview permission.
+        user = UserProfile.objects.get(email='editor@mozilla.com')
+        self.grant_permission(user, 'Addons:ContentReview')
+        AutoApprovalSummary.objects.create(
+            version=self.addon.current_version, verdict=amo.AUTO_APPROVED)
+        self.login_as_editor()
+        response = self.client.post(
+            self.url, {'action': 'confirm_auto_approved'})
+        assert response.status_code == 200
```

Comment:
I am aware that this is the same for regular reviews… but a 200 sounds wrong. Is that POST even possible from the UI?

Reply:
Mmm you're right it does sound wrong. I'll investigate.

Reply:
So, yeah, that POST is not possible from the UI. You get a 200 because we ignore the action. A 400 would be better but that's a refactor I'm not willing to do yet :)